glossary.yml

```diff
@@ -4704,6 +4709,11 @@
     term: የጂኤንዩ የህዝብ ፈቃድ
     def: >
       ሰዎች የለውጦቻቸውን ምንጭ እስካሰራጩ ድረስ ሶፍትዌሮችን እንደገና እንዲጠቀሙ የሚያስችል (ፈቃድ)፡፡
+  es:
+    term: "Licencia Pública GNU"
+    acronym: "GPL"
+    def: >
+    Una [licencia](#license) que permite a las usuarias re-utilizar software siempre y cuando distribuyan el origen de sus cambios 
 
 
 - slug: gradient_boosting
```

Comment:
@GemmaTuron, I think the linter Action is failing because this entry does not have the correct indentation.

Reply:
thanks for the cath @ian-flores !

Reply:
I can't find the indentation problem. The message reads

>   Error: 4761:1 syntax error: could not find expected ':' (syntax)

But I don't see anything odd there

>- slug: gpu